src/QMCDrivers/WFOpt/QMCCostFunctionBatched.cpp

```diff
@@ -337,11 +340,11 @@ void QMCCostFunctionBatched::checkConfigurations()
         // differently for each configuration.  Make sure the same initialization is
         // performed in correlatedSampling.
         *opt_data.get_rng_ptr_list()[ib] = opt_data.get_rng_save();
-        h_list[ib].get().setRandomGenerator(opt_data.get_rng_ptr_list()[ib].get());
+        h_list[ib].setRandomGenerator(opt_data.get_rng_ptr_list()[ib].get());
       }
 
       // Compute distance tables.
-      ParticleSet::flex_update(p_list);
+      ParticleSet::flex_update(p_list, true);
```

Comment:
@markdewing this change I made can be suspicious.

Reply:
I'm testing it now.

Reply:
Yes, that is it.  The default value of skipSK is false.